src/app/core/store/users/users.state.ts

```diff
@@ -49,8 +49,13 @@ export const selectUsersLoadedForOrganization = createSelector(selectUsersState,
 
 export const selectCurrentUser = createSelector(selectUsersState, usersState => usersState.currentUser);
 
+export const selectUserById = (userId: string) => createSelector(selectAllUsers, users => {
+  const usersById = users.filter(user => user.id === userId);
+  return usersById.length > 0 ? usersById.shift() : null;
```

Comment:
This will be inefficient if there are many users in the store. What about using a map instead?
```typescript
export const selectUsersDictionary = createSelector(selectUsersState, usersAdapter.getSelectors().selectEntities);

export const selectUserById = (userId: string) => createSelector(selectUsersDictionary, usersMap => usersMap[userId]);
```

Reply:
This is cool. I did not know about this possibility.